source/windows/file.c

```diff
@@ -13,7 +13,19 @@
 #include <stdio.h>
 #include <windows.h>
 
+static bool s_is_string_non_empty(const struct aws_string *str) {
```

Comment:
Absolutely personal preference and trivial but since every instance of this being used is "!s_is_string_non_empty" I feel like this should be "s_is_string_empty" to make the checks a positive one instead of a double negative.